pyproject.toml

```diff
@@ -198,11 +198,11 @@ keep_full_version = true
 max_supported_python = "3.13"
 
 [tool.pytest.ini_options]
-addopts = "-ra"
+addopts = "-ra --no-cov"
```

Comment:
I tired upgrading Pipfile.lock but because things have started dropping 3.8 a bunch of tests began to fail, but I kept this, since we don't report coverage in the CI anyway, lets not even initialize the plugin.

Reply:
It would be nice though if we did have coverage...